### ISSUE TITLE
ARM support

### ISSUE THREAD

**Issue by saschabrockel** [taylorbobaylor/TrailerDownloader#17]
Hey there. Nice work. Really appreciated. I'm happy that there is a way for me to use it using my Windows PC.

But I have two ARM servers with Plex and the data is on this server. So I access it via SMB and yeah it works but as the download of the trailers is very, very slow I need to keep my PC running day and night, and my electricity bill is rising 😄 

It would be very nice if there would be an option to run the Docker container also on ARM devices because my server is running day and night anyway and only requires a hundredth of the power that my PC needs.

Thank you anyway for your work.

**Comment by taylorbobaylor** [comment]
Glad you're enjoying it! Last time I personally used it, it was very fast. I'll have to take a look again. Not sure about ARM support, I'd have to look into that. 
Work is just crazy right now but I'll try to look at this when I get a chance. 
A lot of this honestly needs to be redone most likely to improve efficio.

**Comment by saschabrockel** [comment]
Okay, interesting. Maybe it is so slow because of my network and not your tool. Because running it on Windows scanning an SMB network share which is on a Linux server with slow bandwidth is maybe problematic. But I got no different choice at the moment. A 30MB trailer needs like 5 minutes. So 5 minutes per movie. And yea that ain't nice.

I also have little free time at the moment and unfortunately no chance to look at how to convert this.

**Comment by saschabrockel** [comment]
Hey, that cannot work because Docker Hub is still telling it is linux/amd64.
[image: image.png]

Am Fr., 18. Feb. 2022 um 19:05 Uhr schrieb Taylor Christian <
***@***.***>:

**Comment by taylorbobaylor** [comment]
Yeah I put the wrong tag and that's why I deleted my comment. I'll continue looking into it. 😉

**Comment by taylorbobaylor** [comment]
Okay, could you try this one?
https://hub.docker.com/repository/docker/taylorbobaylor/arm-test

**Comment by taylorbobaylor** [comment]
Yeah, I love to mess with this stuff when I have the time. I love your ideas and will add them to the next version, whenever that happens to be hahahaha. Don't hold your breath, because it'll definitely be a while before I get the time to seriously refactor all this.
Really glad we got it working for you though!
